lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -264,14 +263,17 @@ def test_linkedin_share_url(self):
         test_url = get_certificate_url(course_id=self.course.id, uuid=self.cert.verify_uuid)
         response = self.client.get(test_url)
         self.assertEqual(response.status_code, 200)
-        params = OrderedDict([
-            ('_ed', '0_0dPSPyS070e0HsE9HNz_13_d11_',),
-            ('pfCertificationName', u'{platform_name} Honor Code Certificate for {course_name}'.format(
-                platform_name=settings.PLATFORM_NAME,
-                course_name=self.course.display_name,
-            ).encode('utf-8'),),
-            ('pfCertificationUrl', self.request.build_absolute_uri(test_url),),
-        ])
+        params = {
+            'name': '{platform_name} Honor Code Certificate for {course_name}'.format(
+                platform_name=settings.PLATFORM_NAME, course_name=self.course.display_name,
+            ).encode('utf-8'),
+            'certUrl': self.request.build_absolute_uri(test_url),
+            # default value from the LinkedInAddToProfileConfigurationFactory company_identifier
+            'organizationId': 1337,
```

Comment:
I don't think it matters since it just ends up getting formatted into a string anyway. /shrug 
So I guess the reason is I find it easier to read as an int vs a number in a string.

Reply:
OK that's fine... I just wanted to make sure we didn't have assumptions in actual code about the format of this theoretically opaque string that we don't control. If it's just test stuff, no worries. I know the model field itself is a text field, which is good.